advanced-list/src/app.js

```diff
@@ -50,30 +45,23 @@ function view(children$, name = '') {
   );
 }
 
-const TickerWrapper = sources => function TickerComponent(color$, id) {
-  const ticker = isolate(Ticker)({DOM: sources.DOM, color: color$});
-  return {
-    DOM: ticker.DOM,
-    action$: ticker.action$.map(a => {
-      a.id = id;
-      return a;
-    }),
-  };
-}
-
 function App(sources) {
-  const tickerProxyAction$ = xs.create();
-  const action$ = intent(sources.DOM, tickerProxyAction$);
-  const tickers$ = model(action$, TickerWrapper(sources));
-  const tickerViews$ = tickers$.map(list => list.map(t => t.DOM));
-  const tickerAction$ = tickers$
-    .map(list => xs.merge(...list.map(t => t.action$)))
-    .flatten();
-  tickerProxyAction$.imitate(tickerAction$);
+  const tickers = Collection(Ticker, sources, {
+    remove$ (tickers, ticker) {
+      return tickers.remove(ticker);
+    }
+  });
```

Comment:
Just to make sure I understood correctly, this `remove$` function was named like this to match the `remove$` in the child sinks object. Right? So Collection matches those.
And what is the semantics of this? "How to map child sinks to reducers for the parent" ?